Application/EdFi.Ods.WebApi/appsettings.json

```diff
@@ -28,6 +28,7 @@
         "PopulatedTemplateDBTimeOutInSeconds": "600",
         "BearerTokenTimeoutMinutes": "30",
         "DefaultPageSizeLimit": 500,
+        "LogRequestResponseContentForSeconds": 0,
```

Comment:
@mjaksn Could you change this setting to minutes. Troubleshooting would take at least several minutes.

Reply:
@vimayya this is now changed to minutes